cmd/crank/install.go

```diff
@@ -75,10 +78,12 @@ func (c *installConfigCmd) Run(k *kong.Context) error {
 	if pkgName == "" {
 		ref, err := name.ParseReference(c.Package)
 		if err != nil {
+			logger.Info(errPkgIdentifier, "error", err)
```

Comment:
It won't have a huge impact right now because we are passing a `NopLogger` if not `--verbose`, but it would likely make more sense for all of these to be `logger.Debug` rather than `logger.Info`. Some good pointers [here](https://crossplane.io/docs/v1.2/contributing/observability_developer_guide.html#in-practice).